medusa/tv/episode.py

```diff
@@ -1702,12 +1702,25 @@ def _format_pattern(self, pattern=None, multi=None, anime_type=None):
                 # cur_name_group_result = cur_name_group.replace(ep_format, ep_string)
                 result_name = result_name.replace(cur_name_group, cur_name_group_result)
 
-        result_name = self.__format_string(result_name, replace_map)
+        parsed_result_name = self.__format_string(result_name, replace_map)
+
+        # With the episode name filenames tend to grow very large. Worst case scenario we even need to add `-thumb.jpg`
+        # to the filename. To make sure we stay under the 255 character limit, we're working with 244 chars, taking into
+        # account the thumbnail.
+        if len(parsed_result_name) > 244 and any(['%E.N' in result_name, '%EN' in result_name, '%E_N' in result_name]):
+            for remove_pattern in ('%E.N', '%EN', '%E_N'):
+                result_name = result_name.replace(remove_pattern, '')
+            # The Episode name can be appended with a - or . in between. Therefor we're removing it.
+            # Creating a clean filename.
+            result_name = result_name.strip('-. ')
```

Comment:
strip() only remove from the beginning or the end of the string. Is it the case here?

Reply:
Yes. Maybe a user puts the episode name a the beginning. At least that was my reasoning for it.